processing file asmFiles/count_days.asm (1 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/fib.asm (2 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/mergesort_17.asm (3 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/mergesort17.asm (4 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/mergesort.asm (5 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/mergesort_ordered.asm (6 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/mult.asm (7 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/mult_procedure.asm (8 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/problem.asm (9 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/search.asm (10 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.assoc.asm (11 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.branch.asm (12 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.branchjump1.asm (13 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.forwarding.asm (14 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.halt.asm (15 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.hazard1.asm (16 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.hazard2.asm (17 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.hazard3.asm (18 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.hazard4.asm (19 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.hazard5.asm (20 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.hazard6.asm (21 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.hit1.asm (22 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.hit2.asm (23 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.jal.asm (24 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.jump.asm (25 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.ldst1.asm (26 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.ldst2.asm (27 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.loadstore.asm (28 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/test.rtype.asm (29 of 29)
diff: memcpu.hex: No such file or directory
processing file asmFiles/count_days.asm (1 of 29)
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2023-10-25 11:47:33.759798180 -0400
+++ memcpu.hex	2023-10-25 11:47:33.744797618 -0400
@@ -13,27 +13,27 @@
 :04000C00FFFFFFFFF4
 :0400210000000001DA
 :0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:040023000000008950
+:040024000000008D4B
+:0400250000000119BD
+:0400260000000121B4
+:04002700000001B123
+:04002800000001BD16
+:040029000000025180
+:04002A00000002616F
+:04002B00000002F9D6
+:04002C000000030DC0
+:04002D00000003A923
+:04002E00000003C10A
+:04002F000000046168
+:040030000000047D4B
+:0400310000000521A5
+:040032000000054184
+:04003300000005E9DB
+:040034000000060DB5
+:04003500000006B908
+:04003600000006E1DF
+:04003700000007912D
 :0403C0000000001623
 :040C4000000000367A
 :00000001FF
processing file asmFiles/mergesort.asm (1 of 1)
--- memsim.hex	2023-10-25 11:48:34.991092146 -0400
+++ memcpu.hex	2023-10-25 11:48:34.974091510 -0400
@@ -149,66 +149,66 @@
 :0401410000000003B7
 :0401420000000004B5
 :0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
+:040144000000000BAC
+:040145000000000CAA
+:0401460000000015A0
+:04014700000000169E
+:04014800000000169D
+:040149000000001999
+:04014A000000001B96
+:04014B000000001D93
+:04014C00000000208F
+:04014D00000000218D
+:04014E000000002984
+:04014F000000003379
+:040150000000003675
+:040151000000003872
+:040152000000003D6C
+:040153000000003D6B
+:040154000000003E69
+:040155000000004264
+:04015600000000475E
+:040157000000004B59
+:040158000000004D56
+:040159000000005052
+:04015A000000005150
+:04015B00000000514F
+:04015C00000000534C
+:04015D000000005A44
+:04015E00000000633A
+:04015F000000006438
+:040160000000000794
+:040161000000000793
+:040162000000000792
+:040163000000000890
+:04016400000000088F
+:04016500000000088E
+:04016600000000098C
+:040167000000000A8A
+:040168000000000F84
+:040169000000001082
+:04016A000000001180
+:04016B000000001779
+:04016C000000001A75
+:04016D00000000206E
+:04016E00000000206D
+:04016F000000002369
+:040170000000002467
+:040171000000002664
+:040172000000002663
+:040173000000002860
+:040174000000002F58
+:040175000000003056
+:040176000000003253
+:040177000000004143
+:040178000000004241
+:04017900000000433F
+:04017A00000000443D
+:04017B000000005927
+:04017C000000005B24
+:04017D000000005C22
+:04017E000000005C21
 :04017F000000006418
-:040C40000000057437
+:040C40000000078722
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/test.assoc.asm (1 of 1)
processing file asmFiles/test.hit1.asm (1 of 1)
processing file asmFiles/test.hit2.asm (1 of 1)
processing file asmFiles/mergesort.asm (1 of 1)
--- memsim.hex	2023-10-25 11:54:49.790138081 -0400
+++ memcpu.hex	2023-10-25 11:54:49.774137482 -0400
@@ -149,66 +149,66 @@
 :0401410000000003B7
 :0401420000000004B5
 :0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
+:040144000000000BAC
+:040145000000000CAA
+:0401460000000015A0
+:04014700000000169E
+:04014800000000169D
+:040149000000001999
+:04014A000000001B96
+:04014B000000001D93
+:04014C00000000208F
+:04014D00000000218D
+:04014E000000002984
+:04014F000000003379
+:040150000000003675
+:040151000000003872
+:040152000000003D6C
+:040153000000003D6B
+:040154000000003E69
+:040155000000004264
+:04015600000000475E
+:040157000000004B59
+:040158000000004D56
+:040159000000005052
+:04015A000000005150
+:04015B00000000514F
+:04015C00000000534C
+:04015D000000005A44
+:04015E00000000633A
+:04015F000000006438
+:040160000000000794
+:040161000000000793
+:040162000000000792
+:040163000000000890
+:04016400000000088F
+:04016500000000088E
+:04016600000000098C
+:040167000000000A8A
+:040168000000000F84
+:040169000000001082
+:04016A000000001180
+:04016B000000001779
+:04016C000000001A75
+:04016D00000000206E
+:04016E00000000206D
+:04016F000000002369
+:040170000000002467
+:040171000000002664
+:040172000000002663
+:040173000000002860
+:040174000000002F58
+:040175000000003056
+:040176000000003253
+:040177000000004143
+:040178000000004241
+:04017900000000433F
+:04017A00000000443D
+:04017B000000005927
+:04017C000000005B24
+:04017D000000005C22
+:04017E000000005C21
 :04017F000000006418
-:040C40000000057437
+:040C40000000078722
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/test.loadstore.asm (1 of 1)
processing file asmFiles/test.ldst1.asm (1 of 2)
processing file asmFiles/test.ldst2.asm (2 of 2)
processing file asmFiles/test.jump.asm (1 of 1)
processing file asmFiles/test.jal.asm (1 of 1)
processing file asmFiles/test.hazard1.asm (1 of 6)
processing file asmFiles/test.hazard2.asm (2 of 6)
processing file asmFiles/test.hazard3.asm (3 of 6)
processing file asmFiles/test.hazard4.asm (4 of 6)
processing file asmFiles/test.hazard5.asm (5 of 6)
processing file asmFiles/test.hazard6.asm (6 of 6)
processing file asmFiles/test.halt.asm (1 of 1)
processing file asmFiles/test.branch.asm (1 of 2)
processing file asmFiles/test.branchjump1.asm (2 of 2)
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2023-10-25 11:58:58.022440968 -0400
+++ memcpu.hex	2023-10-25 11:58:58.008440444 -0400
@@ -13,27 +13,27 @@
 :04000C00FFFFFFFFF4
 :0400210000000001DA
 :0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:040023000000008950
+:040024000000008D4B
+:0400250000000119BD
+:0400260000000121B4
+:04002700000001B123
+:04002800000001BD16
+:040029000000025180
+:04002A00000002616F
+:04002B00000002F9D6
+:04002C000000030DC0
+:04002D00000003A923
+:04002E00000003C10A
+:04002F000000046168
+:040030000000047D4B
+:0400310000000521A5
+:040032000000054184
+:04003300000005E9DB
+:040034000000060DB5
+:04003500000006B908
+:04003600000006E1DF
+:04003700000007912D
 :0403C0000000001623
 :040C4000000000367A
 :00000001FF
processing file asmFiles/mergesort.asm (1 of 1)
--- memsim.hex	2023-10-25 11:59:15.334089660 -0400
+++ memcpu.hex	2023-10-25 11:59:15.317089023 -0400
@@ -149,66 +149,66 @@
 :0401410000000003B7
 :0401420000000004B5
 :0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
+:040144000000000BAC
+:040145000000000CAA
+:0401460000000015A0
+:04014700000000169E
+:04014800000000169D
+:040149000000001999
+:04014A000000001B96
+:04014B000000001D93
+:04014C00000000208F
+:04014D00000000218D
+:04014E000000002984
+:04014F000000003379
+:040150000000003675
+:040151000000003872
+:040152000000003D6C
+:040153000000003D6B
+:040154000000003E69
+:040155000000004264
+:04015600000000475E
+:040157000000004B59
+:040158000000004D56
+:040159000000005052
+:04015A000000005150
+:04015B00000000514F
+:04015C00000000534C
+:04015D000000005A44
+:04015E00000000633A
+:04015F000000006438
+:040160000000000794
+:040161000000000793
+:040162000000000792
+:040163000000000890
+:04016400000000088F
+:04016500000000088E
+:04016600000000098C
+:040167000000000A8A
+:040168000000000F84
+:040169000000001082
+:04016A000000001180
+:04016B000000001779
+:04016C000000001A75
+:04016D00000000206E
+:04016E00000000206D
+:04016F000000002369
+:040170000000002467
+:040171000000002664
+:040172000000002663
+:040173000000002860
+:040174000000002F58
+:040175000000003056
+:040176000000003253
+:040177000000004143
+:040178000000004241
+:04017900000000433F
+:04017A00000000443D
+:04017B000000005927
+:04017C000000005B24
+:04017D000000005C22
+:04017E000000005C21
 :04017F000000006418
-:040C40000000057437
+:040C40000000078722
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/test.rtype.asm (1 of 1)
processing file asmFiles/mergesort.asm (1 of 1)
--- memsim.hex	2023-10-25 14:04:46.332296738 -0400
+++ memcpu.hex	2023-10-25 14:04:46.311295951 -0400
@@ -149,66 +149,66 @@
 :0401410000000003B7
 :0401420000000004B5
 :0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
+:040144000000000BAC
+:040145000000000CAA
+:0401460000000015A0
+:04014700000000169E
+:04014800000000169D
+:040149000000001999
+:04014A000000001B96
+:04014B000000001D93
+:04014C00000000208F
+:04014D00000000218D
+:04014E000000002984
+:04014F000000003379
+:040150000000003675
+:040151000000003872
+:040152000000003D6C
+:040153000000003D6B
+:040154000000003E69
+:040155000000004264
+:04015600000000475E
+:040157000000004B59
+:040158000000004D56
+:040159000000005052
+:04015A000000005150
+:04015B00000000514F
+:04015C00000000534C
+:04015D000000005A44
+:04015E00000000633A
+:04015F000000006438
+:040160000000000794
+:040161000000000793
+:040162000000000792
+:040163000000000890
+:04016400000000088F
+:04016500000000088E
+:04016600000000098C
+:040167000000000A8A
+:040168000000000F84
+:040169000000001082
+:04016A000000001180
+:04016B000000001779
+:04016C000000001A75
+:04016D00000000206E
+:04016E00000000206D
+:04016F000000002369
+:040170000000002467
+:040171000000002664
+:040172000000002663
+:040173000000002860
+:040174000000002F58
+:040175000000003056
+:040176000000003253
+:040177000000004143
+:040178000000004241
+:04017900000000433F
+:04017A00000000443D
+:04017B000000005927
+:04017C000000005B24
+:04017D000000005C22
+:04017E000000005C21
 :04017F000000006418
-:040C40000000057437
+:040C40000000078722
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2023-10-25 14:14:07.308317013 -0400
+++ memcpu.hex	2023-10-25 14:14:07.293316451 -0400
@@ -13,27 +13,27 @@
 :04000C00FFFFFFFFF4
 :0400210000000001DA
 :0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:040023000000008950
+:040024000000008D4B
+:0400250000000119BD
+:0400260000000121B4
+:04002700000001B123
+:04002800000001BD16
+:040029000000025180
+:04002A00000002616F
+:04002B00000002F9D6
+:04002C000000030DC0
+:04002D00000003A923
+:04002E00000003C10A
+:04002F000000046168
+:040030000000047D4B
+:0400310000000521A5
+:040032000000054184
+:04003300000005E9DB
+:040034000000060DB5
+:04003500000006B908
+:04003600000006E1DF
+:04003700000007912D
 :0403C0000000001623
 :040C4000000000367A
 :00000001FF
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/mergesort.asm (1 of 1)
processing file asmFiles/count_days.asm (1 of 29)
processing file asmFiles/fib.asm (2 of 29)
processing file asmFiles/mergesort_17.asm (3 of 29)
processing file asmFiles/mergesort17.asm (4 of 29)
processing file asmFiles/mergesort.asm (5 of 29)
processing file asmFiles/mergesort_ordered.asm (6 of 29)
processing file asmFiles/mult.asm (7 of 29)
processing file asmFiles/mult_procedure.asm (8 of 29)
processing file asmFiles/problem.asm (9 of 29)
processing file asmFiles/search.asm (10 of 29)
processing file asmFiles/test.assoc.asm (11 of 29)
processing file asmFiles/test.branch.asm (12 of 29)
processing file asmFiles/test.branchjump1.asm (13 of 29)
processing file asmFiles/test.forwarding.asm (14 of 29)
processing file asmFiles/count_days.asm (1 of 28)
processing file asmFiles/fib.asm (2 of 28)
processing file asmFiles/mergesort_17.asm (3 of 28)
processing file asmFiles/mergesort17.asm (4 of 28)
processing file asmFiles/mergesort.asm (5 of 28)
processing file asmFiles/mergesort_ordered.asm (6 of 28)
processing file asmFiles/mult.asm (7 of 28)
processing file asmFiles/mult_procedure.asm (8 of 28)
processing file asmFiles/problem.asm (9 of 28)
processing file asmFiles/search.asm (10 of 28)
processing file asmFiles/test.assoc.asm (11 of 28)
processing file asmFiles/test.branch.asm (12 of 28)
processing file asmFiles/test.branchjump1.asm (13 of 28)
processing file asmFiles/test.halt.asm (14 of 28)
processing file asmFiles/test.hazard1.asm (15 of 28)
processing file asmFiles/test.hazard2.asm (16 of 28)
processing file asmFiles/test.hazard3.asm (17 of 28)
processing file asmFiles/test.hazard4.asm (18 of 28)
processing file asmFiles/test.hazard5.asm (19 of 28)
processing file asmFiles/test.hazard6.asm (20 of 28)
processing file asmFiles/test.hit1.asm (21 of 28)
processing file asmFiles/test.hit2.asm (22 of 28)
processing file asmFiles/test.jal.asm (23 of 28)
processing file asmFiles/test.jump.asm (24 of 28)
processing file asmFiles/test.ldst1.asm (25 of 28)
processing file asmFiles/test.ldst2.asm (26 of 28)
processing file asmFiles/test.loadstore.asm (27 of 28)
processing file asmFiles/test.rtype.asm (28 of 28)
